### DISCUSSION TITLE
V0.0.6

### DISCUSSION THREAD

**Discussion by ObuchiYuki** [ObuchiYuki/DevToysMac#6] [Announcements]
# Install

You can use it by downloading `DevToys.zip`, extracting it, and opening it.

# New Features

- Fix Layout Bug
- Fix Typo
- [new] XML Formatter Tab
- [new] Image Converter
- [new] Checksum Generator

<hr /><em>This discussion was created from the release <a href='https://github.com/ObuchiYuki/DevToysMac/releases/tag/0.0.6'>V0.0.6</a>.</em>

**Comment by Ayogua** [comment]
[new] Image Converter
